### DISCUSSION TITLE
How to limit content size in request?

### DISCUSSION THREAD

**Discussion by 0xdeafbeef** [tokio-rs/axum#799] [Q&A] [answered]
Is there any analogue of [content_length_limit](https://docs.rs/warp/latest/warp/filters/body/fn.content_length_limit.html) in warp?

**Comment by davidpdrsn** [comment, marked as answer]
Yep https://docs.rs/axum/latest/axum/extract/struct.ContentLengthLimit.html

**Reply by 0xdeafbeef** [reply]
Thank you for quick answer!

**Comment by Tim-Heartbeat** [comment]
For those using a newer version of Axum, in https://github.com/tokio-rs/axum/pull/1400 ContentLengthLimit was removed in favor of https://docs.rs/axum/latest/axum/extract/struct.DefaultBodyLimit.html
